.changeset/seven-carrots-begin.md

```diff
@@ -0,0 +1,9 @@
+---
+'@vercel/build-utils': minor
+'@vercel/static-build': patch
+'@vercel/hydrogen': patch
+'@vercel/remix-builder': patch
+'@vercel/node': patch
+---
```

Comment:
<!-- Suggestion comments propose improvements to the code. Mention specifically what should be changed and what the benefit would be.  -->
**suggestion (soft-blocking):** remove usage from `packages/next/src/utils.ts`

https://github.com/vercel/vercel/blob/21c700aa93916fbb97c3d2778839b4b67f042b91/packages/next/src/utils.ts#L2901

Reply:
I started to, but it was a bit more complicated. Will ask @ijjk @ztanner to take a look in a follow-up PR.